hdbscan_frontier_filter:
  start_time: 5.5
  obstacle_cost: 40
  info_radius: 0.75
  frontier_clear: 0.01